lib/unique_for_owner_validator.rb

```diff
@@ -13,8 +13,8 @@ def validate(record)
     end
   end
 
-  def unique_fields
-    %i(display_name)
+  def unique_fields_for(record)
```

Comment:
Yeah, I had a better look, don't see anything that can't be done with the standard one I think.